src/Testcontainers.Firestore/FirestoreContainer.cs

```diff
@@ -0,0 +1,21 @@
+namespace Testcontainers.Firestore;
+
+/// <inheritdoc cref="DockerContainer" />
+[PublicAPI]
+public sealed class FirestoreContainer : DockerContainer
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FirestoreContainer" /> class.
+    /// </summary>
+    /// <param name="configuration">The container configuration.</param>
+    /// <param name="logger">The logger.</param>
+    public FirestoreContainer(FirestoreConfiguration configuration, ILogger logger)
+        : base(configuration, logger)
+    {
+    }
+
+    public void SetEmulatorHost()
+    {
+        Environment.SetEnvironmentVariable("FIRESTORE_EMULATOR_HOST", $"{Hostname}:8080/");
```

Comment:
I took a brief look and noticed the environment variable. Do you think it is possible to change it to support the `FirestoreDbBuilder` properties:

```csharp
var firestoreDbBuilder = new FirestoreDbBuilder();
firestoreDbBuilder.Endpoint = "";
firestoreDbBuilder.JsonCredentials = "";
```

Depending on the environment variable might limit us in running multiple containers at once.